mock-core-configs/etc/mock/templates/centos-stream-9.tpl

```diff
@@ -0,0 +1,48 @@
+config_opts['chroot_setup_cmd'] = 'install tar gcc-c++ redhat-rpm-config redhat-release which xz sed make bzip2 gzip gcc coreutils unzip shadow-utils diffutils cpio bash gawk rpm-build info patch util-linux findutils grep'
+config_opts['dist'] = 'el9'  # only useful for --resultdir variable subst
+config_opts['releasever'] = '9'
+config_opts['package_manager'] = 'dnf'
+config_opts['extra_chroot_dirs'] = [ '/run/lock', ]
+config_opts['bootstrap_image'] = 'quay.io/centos/centos:stream9'
+
+config_opts['dnf_vars'] = { 'stream': '9-stream',
+                            'contentdir': 'centos',
+                          }
```

Comment:
These variables are specific to the way 8-stream is mirrored.  We haven't yet settled on a mirror strategy for 9-stream.  I would recommend removing them until we know that they are necessary.